### PR TITLE
GIX-2102: Remove default firstColumnHeader

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove default value for `firstColumnHeader` to make it configurable from outside always.

# Changes

* Remove default value from `firstColumnHeader` in TokensTable.svelte
* Add the old default value in the usages of TokensTable without the prop `firstColumnHeader`.

# Tests

* Remove a test for the default column header in TokensTable.
* Add tests in the tokens route to check the header's title, both logged in and out.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3829** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3829?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3829?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/routes/app/tokens/page.spec.ts [hunk 1]
```diff
@@ -145,6 +145,13 @@ describe("Tokens route", () => {
         expect(await po.hasTokensPage()).toBe(true);
       });
 
+      it("renders 'Accounts' as tokens table first column", async () => {
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/tests/routes/app/tokens/page.spec.ts]: Thanks! Yes

File: frontend/src/tests/routes/app/tokens/page.spec.ts [hunk 2]
```diff
@@ -221,6 +228,13 @@ describe("Tokens route", () => {
         expect(await po.hasTokensPage()).toBe(false);
       });
 
+      it("renders 'Accounts' as tokens table first column", async () => {
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/tests/routes/app/tokens/page.spec.ts]: Thanks, done!

**Review by dskloetd — APPROVED** [review]

**Pull request merged**